### ISSUE TITLE
You'll vs We'll

### ISSUE THREAD

**Issue by hackettka** [DHRI-Curriculum/git#31]
You'll and we'll seems to be used interchangeably throughout the lesson - such as you'll be using git to .. and we'll be using markdown to .. in the different sections. I started to use you'll, and then found more instances of we'll, so i stopped. i'm not sure which makes more sense. you'll maybe? can do either, but think it should be universally implemented. here and across the curriculum.

**Comment by kallewesterling** [comment]
@lmrhody This seems like an important point for us to take to heard about the editorial style for the entire syllabus.

I think it also ties in with @smorello87's question about style in https://github.com/DHRI-Curriculum/command-line/issues/60 so I wanted to make the connection here between the two.

The [original style guide](https://github.com/DHRI-Curriculum/guide/blob/master/sections/style-guide.md) does not contain anything about language that should be used for the workshops.

**Comment by kallewesterling** [comment]
I talked this over with @lmrhody a little while ago and forgot to close out this issue. Sorry about that @hackettka—what we decided was that it's up to the fellow to let their personal style shine through in the language used in each workshop. I'm going to go ahead and close the issue for now, but will make a note in my own reflections on future directions for the project, once this summer's push is over, and we want to think about what comes next, for the DHRI and the website/developing new workshops, etc.
